lib/resources.js

```diff
@@ -83,16 +88,19 @@ const SEARCH_SCOPES = {
       'seriesStatement.folded',
       'contentsTitle.folded',
       'donor.folded',
-      'parallelTitle.folded',
+      'parallelTitle.folded^5',
       'parallelTitleDisplay.folded',
       'parallelSeriesStatement.folded'
+      'parallelTitleAlt.folded',
+      'parallelCreatorLiteral.folded',
+      'parallelUniformTitle'
     ]
   },
   contributor: {
-    fields: ['creatorLiteral^4', 'creatorLiteral.folded^2', 'contributorLiteral.folded']
+    fields: ['creatorLiteral^4', 'creatorLiteral.folded^2', 'contributorLiteral.folded', 'parallelCreatorLiteral', 'parallelContributorLiteral']
```

Comment:
I believe these will need to use the `.folded` fields since those are analyzed by icu, which has been shown to tokenize non-Roman scripts seemingly pretty well